code/modules/mob/living/carbon/human/dummy.dm

```diff
@@ -117,6 +117,9 @@ INITIALIZE_IMMEDIATE(/mob/living/carbon/human/dummy)
 	if(is_creating)
 		fully_replace_character_name(real_name, "John Doe")
 
+/mob/living/carbon/human/consistent/domutcheck()
+	return // We skipped adding any mutations so this runtimes
```

Comment:
This won't screw with using the consistent dummy in other unit tests, right? (i.e.: this should have been implemented at the start due to certain assumptions about the DNA with the dummy, but wasn't)

Reply:
In my investigation it seems that this proc is _mostly_ called manually, and not by any of the unit tests or anything that was being unit tested.
I could not find anywhere which was setting some actual mutations and _then_ calling it, and if not doing that it would have runtimed. So as far as I could ascertain it shouldn't have side effects on other tests.
The alternative to this is actually giving the dummy a predefined list of dormant mutations in its DNA, but it's never needed it before.